domainobjs/ts/__tests__/DomainObjs.test.ts

```diff
@@ -1,286 +0,0 @@
-        it("Should convert a JSON file from snarkjs to a VerifyingKey", () => {
-            const file = path.join(__dirname, "test_vk.json");
-            const j = fs.readFileSync(file).toString();
-            const d = JSON.parse(j);
-            const vk = VerifyingKey.fromJSON(j);
-
-            expect(d.vk_alpha_1[0]).to.eq(vk.alpha1.x.toString());
-            expect(d.vk_alpha_1[1]).to.eq(vk.alpha1.y.toString());
-
-            expect(d.vk_beta_2[0][0]).to.eq(vk.beta2.x[1].toString());
-            expect(d.vk_beta_2[0][1]).to.eq(vk.beta2.x[0].toString());
-            expect(d.vk_beta_2[1][0]).to.eq(vk.beta2.y[1].toString());
-            expect(d.vk_beta_2[1][1]).to.eq(vk.beta2.y[0].toString());
-
-            expect(d.vk_gamma_2[0][0]).to.eq(vk.gamma2.x[1].toString());
-            expect(d.vk_gamma_2[0][1]).to.eq(vk.gamma2.x[0].toString());
-            expect(d.vk_gamma_2[1][0]).to.eq(vk.gamma2.y[1].toString());
-            expect(d.vk_gamma_2[1][1]).to.eq(vk.gamma2.y[0].toString());
-
-            expect(d.vk_delta_2[0][0]).to.eq(vk.delta2.x[1].toString());
-            expect(d.vk_delta_2[0][1]).to.eq(vk.delta2.x[0].toString());
-            expect(d.vk_delta_2[1][0]).to.eq(vk.delta2.y[1].toString());
-            expect(d.vk_delta_2[1][1]).to.eq(vk.delta2.y[0].toString());
-
-            expect(d.IC.length).to.eq(vk.ic.length);
-            for (let i = 0; i < d.IC.length; i++) {
-                expect(d.IC[i][0]).to.eq(vk.ic[i].x.toString());
-                expect(d.IC[i][1]).to.eq(vk.ic[i].y.toString());
-            }
-        });
-    });
-
-    describe("State leaves", () => {
-        it("The serialize() and unserialize() functions should work correctly", () => {
-            const stateLeaf = new StateLeaf(
-                pubKey,
-                BigInt(123),
-                BigInt(1231267)
-            );
-
-            const serialized = stateLeaf.serialize();
-            const unserialized = StateLeaf.unserialize(serialized);
-
-            expect(unserialized.voiceCreditBalance.toString()).to.eq(
-                stateLeaf.voiceCreditBalance.toString()
-            );
-        });
-    });
-
-    describe("Keypairs", () => {
-        it("the Keypair constructor should generate a random keypair if not provided a private key", () => {
-            const k1 = new Keypair();
-            const k2 = new Keypair();
-
-            expect(k1.equals(k2)).to.be.false;
-
-            expect(k1.privKey.rawPrivKey).not.to.eq(k2.privKey.rawPrivKey);
-        });
-
-        it("the Keypair constructor should generate the correct public key given a private key", () => {
-            const rawKeyPair = genKeypair();
-            const k = new Keypair(new PrivKey(rawKeyPair.privKey));
-            expect(rawKeyPair.pubKey[0]).to.eq(k.pubKey.rawPubKey[0]);
-            expect(rawKeyPair.pubKey[1]).to.eq(k.pubKey.rawPubKey[1]);
-        });
-
-        it("PrivKey.serialize() and unserialize() should work correctly", () => {
```

Comment:
Looks like some tests (e.g. this one) is no longer present - why?

I thought the main goal here was separating the tests into independent files, not actually changing the tests. Is that right?

Reply:
Ah, I see - you also created `privateKey` & `publicKey` test files - nice 😄